activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakManagementService.java

```diff
@@ -242,19 +242,18 @@ public List<SecurityResponseRepresentation> getApplicationPermissions(String app
 
     @Override
     public List<User> findUsersByGroupName(String groupName) {
-        List<User> users = new ArrayList<>();
-        if (!StringUtils.isEmpty(groupName)) {
-            Optional<Group> groupOptional = findGroupStrictlyEqualToGroupName(groupName);
-            groupOptional.ifPresent(group ->
-                    users.addAll(getUsersByGroupId(group.getId())));
-        }
-        return users;
+        Group groupFound = findGroupStrictlyEqualToGroupName(groupName);
+        return getUsersByGroupId(groupFound.getId());
     }
 
-    private Optional<Group> findGroupStrictlyEqualToGroupName(String groupName) {
-        return findGroups(groupName).stream()
-            .filter(group -> groupName.equals(group.getName()))
-            .findFirst();
+    private Group findGroupStrictlyEqualToGroupName(String groupName) {
+        return
+            Optional.ofNullable(groupName)
+                .map(g -> findGroups(g).stream())
+                .orElse(Stream.empty())
```

Comment:
is it required?

Reply:
Yes, It is the only way that I have found